src/pages/homepage/Homepage.jsx

```diff
@@ -0,0 +1,20 @@
+import "./Homepage.css";
+import { Footer } from "../../components";
+import Hero from "./components/hero/Hero";
+import Categories from "./components/categories/Categories";
+import Products from "./components/products/Products";
+
+
+
+export default function Homepage() {
+  return (
+    <>
+       <Hero /> 
+      <main className="homepage-main-section">
+       <Categories /> 
+       <Products /> 
+      </main>
```

Comment:
You should rename this as Main, all the components in react should start with a capital letter.

Reply:
Hey atul, thanks for your feedback but main here is a plain html element and not a React Component, am I missing something please revert back if it is so.

Reply:
Oh sorry! I missed that part.